### DISCUSSION TITLE
李嘉诚汕头大学毕业典礼的节录

### DISCUSSION THREAD

**Discussion by ergatea** [RoadSideUniversity/ClubHouse2024#34] [General]
我明年90岁啦，一生志在千里，也知似水流年；我年轻过，历尽困难试炼，我深刻知道成长之路是非常不容易的；在高增长机遇巨浪中，愚人见石，智者见泉。

因循的并发症是不思不想和无感无知，在人工智能世代中肯定过不了关；驾浪者的基本功，时时刻刻要灵敏、快知快明，要有独立思考悟力、能运用想象，把现实、数据、信息合组成新。

- >愚人只知道‘为’（to do），智者有愿力，把‘为’（to do）变‘成为’（to be）。
‘愿力一族’是如何修炼？如何处世？如何存在？

愚人常常抱怨，变得墨守成规是被逼出来的，被制度营役、被繁文缛节捆绑、被不可承受的期望压至透不过气；他们渴望‘赢在起跑线上’，希望有个富爸加上天赋的优越组合，认为‘人能弘道’、改变尘世复杂和无可奈何的扭曲太负重，‘道能弘人’肯定更舒服。

这样的心态，他们已‘输在起跑线上’。传统中国智慧告诫我们命与运是互动交织的，拥有一切，也可以一无所有。懂得‘善择’才是打造自己命运的保证。而命运大赢家的梦幻 DNA 组合，是科学心智与艺术心灵的觉醒，才可把潜能修炼为出众的人生。性格基础是意志力，自律的坚持和创意潜力相形相塑，才可达致拥有挪移心外喧哗的处世心力。

自律是铁杵成针的意志功夫。每个希望成为大舞蹈家的人，每天面镜，并非顾影自怜，而是不怕疲惫、不怕痛苦，一而再，再而三，修正追求举重若轻的完美，技巧内化自我之中。走到台前，‘身与物化，意到图成’。

我今天为什么选择舞蹈幻灯为背景？诗人叶慈的提问，一矢中的，‘怎样才能从舞蹈之中辨别真正的舞者？’舞蹈家个性魅力触动观众，凝住一瞬永恒，艺术映照人生，启迪感召每一个人逾越艰难超越局限，追求更高的水平，开拓无限的可能。
最后，我想和大家分享，去年我和一位多年在汕头大学服务的老师的对话，当我感谢他多年孜孜不倦培育学生时， 老师引用王阳明一语：‘千圣皆过影，良知乃吾师。’良知是成就尊严和有存在意义的明灯。

各位同学，道力之限，要靠愿力突破，我知道，你们一定会怀着谦恭、感恩的心，以信心和想象力追求一个开放、进步的世界，建立一个关怀的社会，成为真正的舞者，今天你以汕大为荣，明天汕大必将以你为荣。再次祝贺你们！谢谢大家。”

**Comment by Lewisrxliu** [comment]
👍”良知乃吾师”， 洞见深刻，感谢分享！

**Comment by Lewisrxliu** [comment]
致敬🫡 查理芒格 Charlie Munger （1924-2023）
![image](https://github.com/RoadSideUniversity/ClubHouse2023Nov/assets/128250203/c3beaf0c-c2a4-46d6-a8b7-c6190a896765)
